routers/api/v1/api.go

```diff
@@ -74,7 +74,7 @@ import (
 	api "code.gitea.io/sdk/gitea"
 
 	"github.com/go-macaron/binding"
-	macaron "gopkg.in/macaron.v1"
```

Comment:
This must have happened automatically via GoLand. Let me know if you want me to revert it, but it must work without it.